src/helpers.js

```diff
@@ -0,0 +1,83 @@
+//!Defines two helper functions.
+
+/*
+ * c
+ * https://github.com/rumpl/c
+ *
+ * Copyright (c) 2012 Djordje Lukic
+ * Licensed under the MIT license.
+ */
+
+"use strict";
+const underscore = require("underscore");
```

Comment:
This one is not used any more, it can also be removed from the package.json file :)